joomla/joomla-cms
# Search Tools (from JLayouts) implemented in com_checkin

#### Description

In move to normalize the search/filter/order/limit bar (aka `searchtools`) in all administrator list views, and as other PR's have made for other list component/views, this PR adds JLayouts `searchtools` to com_checkin only view (checkin) and replacing the old HTML code.

Also:
- Some HTML code corrected (table blocks order)
- Resolves bug in pagination when set to "All" (none appeared)
- Other minor improvements
##### Isis template
###### Before patch

![com_checkin_isis_before](https://cloud.githubusercontent.com/assets/9630530/12184036/b56231b2-b58a-11e5-97ef-80ffe50530f2.png)
###### After patch

![com_checkin_isis_after](https://cloud.githubusercontent.com/assets/9630530/12184035/b2f21550-b58a-11e5-9c75-ab339e26d1b2.png)
##### Hathor template
###### Before & after patch (no change)

![com_checkin_hathor_after](https://cloud.githubusercontent.com/assets/9630530/12184033/af0b5a46-b58a-11e5-86e4-9fa70315fedb.png)
#### How to test
1. Edit some content and don't save or close so they are in not checked status
2. Go to com_checkin ("System" -> "Global check-in").
3. Apply the patch
4. Go again to com_checkin ("System" -> "Global check-in").
5. Test the Search Tools (Search field, ordering and limit).
6. View hathor template too.
#### Observations

Suggestions and code revisions are welcome.
#### Tags

administrator, checkin, search tools, jlayouts, list views, normalization
